circle.yml

```diff
@@ -7,8 +7,21 @@ general:
   artifacts:
```

Comment:
I really like Circle, and it's working for now, so I'm inclined to stick with that.